src/Impostor.Api/Innersloth/Customization/PlayerOutfit.cs

```diff
@@ -2,19 +2,67 @@ namespace Impostor.Api.Innersloth.Customization
 {
     public class PlayerOutfit
     {
-        public ColorType Color { get; internal set; } = (ColorType)(-1);
+        private string _playerName = string.Empty;
+        private ColorType _color = (ColorType)(-1);
```

Comment:
InnerSloth assigns 0 red here for initial data

Reply:
Hmm, fair point, this is a bit ugly indeed. We can't however just set this to 0 as that'd interfere with the used color anticheat rule. What do you think about making this nullable instead, and serializing this as 0?

Considering that it was set to `-1` already in the old version, do you think this commit should be modified to add this change or should this be a separate PR?

Reply:
i think putting -1 is better than what innersloth does

Reply:
Innersloth assigns -1 here too (see NetworkedPlayerInfo.PlayerOutfit.ColorId). Resolving this discussion.